CMakeLists.txt

```
@@ -6,7 +6,7 @@ project(SlicerOpenLIFU)
 # Extension meta-information
 set(EXTENSION_HOMEPAGE "https://github.com/OpenwaterHealth/SlicerOpenLIFU")
 set(EXTENSION_CONTRIBUTORS "Ebrahim Ebrahim (Kitware), Peter Hollender (Openwater), Sam Horvath (Kitware), Andrew Howe (Kitware), Sadhana Ravikumar (Kitware), Brad Moore (Kitware)")
-set(EXTENSION_DESCRIPTION "A 3D Slicer extension for Openwater's OpenLIFU (Low Intensity Focused Ultrasound) research platform.")
+set(EXTENSION_DESCRIPTION "A 3D Slicer extension for Openwater’s OpenLIFU (Low Intensity Focused Ultrasound) research platform. Licensed under AGPL (a strong copyleft license that may limit use in proprietary or clinical products).")
```

```suggestion
set(EXTENSION_DESCRIPTION "A 3D Slicer extension for Openwater’s OpenLIFU (Low Intensity Focused Ultrasound) research platform. Licensed under AGPL (a strong copyleft license that may impose restrictions on combined works).")
```

(just copying language from the note in README)